lisa/util/shell.py

```diff
@@ -137,12 +137,13 @@ def try_connect(
     timeout_start = time.time()
     while time.time() < timeout_start + ssh_timeout:
         try:
+            pkey = paramiko.RSAKey.from_private_key_file(connection_info.private_key_file)
```

Comment:
What's the reason to change it? It looks the logic works for RSA only.

Reply:
Maybe it relates to OpenSSL version. It happens on some old CentOS 6.x distros as well, because paramiko doesn't fully support some old SSH behaviors.